### DISCUSSION TITLE
Repeater Setting

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#11847] [Help] [answered]
Hey ...,
  I have used Repeater(Component) but it repeats multiple pages. What do I have to do to keep it down to just one page?
![image](https://github.com/Budibase/budibase/assets/93240913/d21d3f09-95f7-4247-a44f-50cd6158a2ca)
  
Thank you,

**Comment by andz-bb** [comment, marked as answer]
hey @LTCTHITA 

The Repeater component repeats all of its children for every item coming from the data source you have chosen. If you are using the Data Provider as the Repeater's data source, you should limit it to 1 result and apply a relevant filter to it (by the Search Tag no field value, for example). All of the settings you need will be on the Data Provider. Hope this helps!

<img width="311" alt="image" src="https://github.com/Budibase/budibase/assets/110921612/ee970178-9bd1-480a-9f75-8b7e6ee0328a">
